app/assets/stylesheets/application.scss

```diff
@@ -60,22 +54,17 @@ $white: #ffffff;
   margin-top: 0;
 }
 
-.home-form {
-  color: $white;
-}
-
 .entries {
   background-color: $white;
+  padding: 5px;
```

Comment:
Properties should be ordered background-color, border-bottom, list-style, padding, text-align